django-s3-file-field-client/src/client.ts

```diff
@@ -19,6 +19,17 @@ interface UploadedPart {
   size: number;
   etag: string;
 }
+
+interface ProgressEvent {
```

Comment:
Since we're defining our own interface here, I think we can do better than the upstream [`ProgressEvent` API](https://developer.mozilla.org/en-US/docs/Web/API/ProgressEvent). For instance, `lengthComputable` probably should be removed, since we know we can always report it here.